model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -117,6 +117,25 @@ class LicenseInfoResolver(
                         )
                     )
                 }
+
+                licenseInfo.declaredLicenseInfo.copyrightHolders.takeIf { it.isNotEmpty() }?.let {
+                    locations.add(
```

Comment:
Use `+=` instead of `add()` (I'm fixing this for existing code in https://github.com/oss-review-toolkit/ort/pull/5416).

Reply:
Done